packages/e2e/e2e_macos/macos/e2e_macos.podspec

```diff
@@ -9,11 +9,10 @@ Pod::Spec.new do |s|
 Runs tests that use the flutter_test API as integration tests on macOS.
```

Comment:
@franciscojma86  Is e2e_macos published anywhere?

Reply:
I don't think so. IIRC, there is still no support for native tests on macOS